QA_Robot_Practice/ui/Browser.py

```diff
@@ -0,0 +1,56 @@
+from selenium import webdriver
```

Comment:
ui packedge is not necessary because selenium2library should be used in Pages
as
 import Selenium2Library